src/plugins/intel_gpu/src/kernel_selector/cl_kernels/gemm_mmad_int8.cl

```diff
@@ -469,10 +469,6 @@ KERNEL(gemm_mmad_int8)(
     }
 
 #if TILE_NUM == 2
-#if HAS_FUSED_OPS && FUSED_OPS_CAN_USE_PRELOAD
-    FUSED_OPS_PRELOAD;
-#endif
```

Comment:
Could you explain why it is removed?

Reply:
To fix build failure for redefinition issue, but original expectation is blurry, so remove it and let's take another pr.